integration_tests/src/main/python/string_test.py

```diff
@@ -771,6 +771,23 @@ def test_regexp_extract_idx_0():
                 'regexp_extract(a, "^([a-d]*)[0-9]*([a-d]*)\\z", 0)'),
         conf=_regexp_conf)
 
+def test_regexp_whitespace():
+    gen = mk_str_gen('[abcd]\t\n{1,3} [0-9]\n {1,3}\x0b\t[abcd]\r\f{1,3}')
```

Comment:
I would prefer to see some non-whitespace characters at the end of the generated string so that we also have tests covering carriage return and form feeds within the string rather than just at the end of the string.

Reply:
Done.